typescript/introduction/tsc-basics/tsc-assigning-types.md

```diff
@@ -0,0 +1,91 @@
+---
+author: kapnobatai136
+
+type: normal
+
+category: must-know
+
+aspects:
+  - introduction
+
+---
+
+# Assigning Types
+
+---
+## Content
+
+Assigning types is one of the main features of TypeScript.
+
+```ts
+function hello(who: string): void {
+  console.log('Hello ' + who);
+}
+hello('Enki'); // Hello Enki
+hello(5); // TypeError
+```
+
+In this example, the `who` argument always has to be a string, and the `hello` function can't return anything (from the `function hello(): void` syntax). Notice that when we try to call the function using a number, we get a type error. The same would happen if we tried to include a `return` statement in the function.
```

Comment:
```suggestion
In this example, the `who` argument always has to be a string, and the `hello` function can't return anything (because the return type is `void`). 

Notice that when we try to call the function using a number, we get a type error. The same would happen if we tried to include a `return` statement in the function.
```